exberry_adapter/test/daml/Tests/Matching.daml

```diff
@@ -0,0 +1,98 @@
+module Tests.Matching where
```

Comment:
I believe there used to be an `integrationTesting` model in this repo (I'm not sure when or why it was removed). This is similar, but runs the full test automatically. The test is basic but could be extended, possibly for a larger volume stress test.

Reply:
It was removed as the code of the `integrationTesting` went out of sync very quickly and it was a very manual task to execute the testing in that you had to modify the code for almost each test followed by calling the various functions available in the correct order. The purpose of the `integrationTesting` was to test our daml models with exberry before the UI was finished.